tests/e2e/specs/add-edit-category.test.js

```diff
@@ -0,0 +1,227 @@
+import { 
+	visitAdminPage,
+} from "@wordpress/e2e-test-utils";
+import { addQueryArgs } from "@wordpress/url";
+
+async function deleteAllCategories() {
+	const categoriesPageQuery = addQueryArgs( '', {
+		taxonomy: 'category',
+	} );
+	await visitAdminPage( 'edit-tags.php', categoriesPageQuery );
+
+	// Wait for the categories rows to appear
+	await page.waitForSelector( '#the-list tr' );
+
+	const allCategoriesRows = await page.$$( '#the-list tr' );
+	
+	// If there is more than one category row, delete all the categories
+	if( allCategoriesRows.length > 1 ) {
+		await page.click( '[id^=cb-select-all-]' );
+		await page.select( '#bulk-action-selector-top', 'delete' );
+		await page.focus( '#doaction' );
+		await page.keyboard.press( 'Enter' );
+	}
```

Comment:
Might be good to check that all categories have actually been deleted by doing something like `await page.waitForSelector( '#the-list tr:only-child' );` at the end of the function.